charts/portainer/templates/deployment.yaml

```diff
@@ -25,9 +25,11 @@ spec:
       {{- end }}
       serviceAccountName: {{ include "portainer.serviceAccountName" . }}
       volumes:
+        {{- if .Values.persistence.enabled }}
```

Comment:
Just wonder that if both `.Values.persistence.enabled` and `.Values.tls.existingSecret` are false, the deployment manifest will only have the key `volumes` without value. Is it okay?
```
...
    spec:
      nodeSelector:
        {}
      serviceAccountName: portainer-sa-clusteradmin
      volumes:
      containers:
        - name: portainer
          image: "portainer/portainer-ce:latest"
          imagePullPolicy: Always

...
```